utils/jmxfiles.py

```diff
@@ -0,0 +1,109 @@
+# std
+import logging
+import os
+import tempfile
+import time
+
+# 3rd party
+import yaml
+
+# datadog
+from config import _windows_commondata_path
+from util import yDumper
+from utils.pidfile import PidFile
+from utils.platform import Platform
+
+
+log = logging.getLogger(__name__)
+
+
+class JMXFiles(object):
```

Comment:
I don't get the point of writing a class with only class methods inside (at least not in Python :) ). Couldn't there just be static methods. Then if you import it this way : `from utils import jmxfiles` you could just call `jmxfiles.get_status_file_path()`. Wouldn't it make more sense.